app/Http/Controllers/Apis/Protected/Summit/OAuth2SummitPromoCodesApiController.php

```diff
@@ -758,4 +759,60 @@ function ($attribute, $value, $fail) use ($filter) {
             return $this->ok();
         });
     }
+
+    /**
+     * @param $summit_id
+     * @return \Illuminate\Http\JsonResponse|mixed
+     */
+    public function sendSponsorPromoCodes($summit_id)
+    {
+        return $this->processRequest(function () use ($summit_id) {
+
+            if (!Request::isJson()) return $this->error400();
+
+            $summit = SummitFinderStrategyFactory::build($this->summit_repository, $this->resource_server_context)->find($summit_id);
+            if (is_null($summit)) return $this->error404();
```

Comment:
u need to check authz here is this endpoint is meant to be used by summits admins